src/sidebar/util/test/group-organizations-test.js

```diff
@@ -0,0 +1,158 @@
+'use strict';
+
+var groupsByOrganization = require('../group-organizations');
```

Comment:
We're using `const` and `let` elsewhere in the file. Any reason not to use it here too?

My plan was that at some point we would run an automated fixer over existing code to make the style uniform.